src/components/ConfirmTx/ConfirmTx.test.tsx

```diff
@@ -3,10 +3,11 @@ import {render} from '@testing-library/react-native'
 import React from 'react'
 import {Provider} from 'react-redux'
 
-import configureStore from '../../../legacy/helpers/configureStore.js'
+import configureStore from '../../legacy/configureStore.js'
 import {HaskellShelleyTxSignRequest} from '../../yoroi-wallets/index.js'
 
-global.window = {}
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+;(global as any).window = {}
```

Comment:
```suggestion
(global as any).window = {}
```

Reply:
the semicolon is added automatically, it cant be removed